diff --git a/node_modules/@octokit/webhooks/dist-node/index.js b/node_modules/@octokit/webhooks/dist-node/index.js
index a231414..55cab64 100644
--- a/node_modules/@octokit/webhooks/dist-node/index.js
+++ b/node_modules/@octokit/webhooks/dist-node/index.js
@@ -296,25 +296,7 @@ function verifyAndReceive(state, event) {
 
 const debugWebhooks = debug.debug("webhooks:receiver");
 function middleware(state, request, response, next) {
-  if (isntWebhook(request, {
-    path: state.path
-  })) {
-    // the next callback is set when used as an express middleware. That allows
-    // it to define custom routes like /my/custom/page while the webhooks are
-    // expected to be sent to the / root path. Otherwise the root path would
-    // match all requests and would make it impossible to define custom rooutes
-    if (next) {
-      next();
-      return;
-    }
-
-    debugWebhooks(`ignored: ${request.method} ${request.url}`);
-    response.statusCode = 404;
-    response.end("Not found");
-    return;
-  }
-
-  const missingHeaders = getMissingHeaders(request).join(", ");
+    const missingHeaders = getMissingHeaders(request).join(", ");
 
   if (missingHeaders) {
     const error = new Error(`[@octokit/webhooks] Required headers missing: ${missingHeaders}`);
